### DISCUSSION TITLE
Feature request | Add shortcuts / URL Scheme / Widget to turn on/off DNS

### DISCUSSION THREAD

**Discussion by MoienBowen** [kkebo/DNSecure#57] [Ideas]
Hi,
In some case, I need to switch back to the default DNS.
If you can add the shortcuts, URL Schemes and/or widgets to turn on/off the DNS setting, even to switch different DNS, that’s could be excellent.

**Comment by kkebo** [comment]
This is impossible because of the same reason as #71.
